glean-core/android/src/main/java/mozilla/telemetry/glean/net/Upload.kt

```diff
@@ -43,14 +43,18 @@ internal class UploadBody(
     }
 }
 
-@Structure.FieldOrder("tag", "upload")
```

Comment:
Why was this removed?

Reply:
See the commit: a union hardly as an order, only one field is "active" at a time.
(The underlying `Union` inherits from `Structure` and just has a default implementation for `FieldOrder` to satisfy the class hierarchy)